### PR TITLE
 New modal layout for creating events

### PR DESCRIPTION
**Pull request by Luiz-FS**
<p><b>Feature/Bug description:</b>  Create a new layout for the modal event creation.</p>

screenshot from 2017-12-21 10-14-28

screenshot from 2017-12-21 10-14-47
screenshot from 2017-12-21 10-15-11
screenshot from 2017-12-21 10-15-42
screenshot from 2017-12-21 10-16-03
screenshot from 2017-12-21 10-16-17
screenshot from 2017-12-21 10-16-59
screenshot from 2017-12-21 10-17-15
screenshot from 2017-12-21 10-17-27

<p><b>Solution:</b> Using the angular directives and the pre-defined screens by design, I turned the new layout to the event creation modal.</p>

<p><b>TODO/FIXME:</b> n/a</p>

### CODE REVIEW

File: frontend/event/eventController.js [hunk 1]
```diff
@@ -204,6 +212,28 @@
...
         };
 
+        dialogCtrl.removeUrl = function(url, urlList) {
+            _.remove(urlList, function(element) {
+             return element === url;
+           });    
+        };
+
+        dialogCtrl.changeUrlLink = function(urlLink, urlList) {
+            console.log(dialogCtrl.event);
```
Comment (andreldsa) [inline frontend/event/eventController.js]: console.log

File: frontend/event/eventController.js [hunk 2]
```diff
@@ -204,6 +212,28 @@
             }
         };
 
+        dialogCtrl.removeUrl = function(url, urlList) {
+            _.remove(urlList, function(element) {
```
Comment (andreldsa) [inline frontend/event/eventController.js]: Strange indentation.

File: frontend/event/eventController.js [hunk 3]
```diff
@@ -322,7 +412,23 @@
             }
         }
 
+        function addLinks(event) {
+            let videoUrls = _.filter(dialogCtrl.videoUrls, e => e.url !== '');
```
Comment (andreldsa) [inline frontend/event/eventController.js]: Change the variable 'e' name.

File: frontend/event/eventController.js (hunk 2 above)
Comment (Luiz-FS) [reply frontend/event/eventController.js]: Done.

File: frontend/event/eventController.js [hunk 4]
```diff
@@ -204,6 +212,27 @@
...
+        dialogCtrl.addUrl = function(urlList){
+            const hasEmptyUrl = dialogCtrl.getEmptyUrl(urlList) !== undefined;
+            urlList.length < 10 && !hasEmptyUrl && urlList.push(angular.copy(emptyUrl));
+        };
+
+        dialogCtrl.getEmptyUrl = function(urlList){
+            return _.find(urlList, emptyUrl);
+        };
+
+
```
Comment (andreldsa) [inline frontend/event/eventController.js]: rlb

**Pull request merged**